tests/hlsl-intrinsic/helper-lane.slang

```diff
@@ -0,0 +1,13 @@
+//TEST:SIMPLE(filecheck=HLSL): -entry main -target hlsl -stage fragment
+//TEST:SIMPLE(filecheck=GLSL): -entry main -target glsl -stage fragment
+//TEST:SIMPLE(filecheck=METAL): -entry main -target metal -stage fragment
+//TEST:SIMPLE(filecheck=SPIRV): -entry main -target spirv -stage fragment
+
+float4 main() : SV_Target
+{
+    //HLSL:IsHelperLane()
+    //GLSL:gl_HelperInvocation()
+    //METAL:simd_is_helper_thread()
+    //SPIRV:OpIsHelperInvocationEXT
```

Comment:
GLSL should check if: `GL_EXT_demote_to_helper_invocation` is enabled

SPIRV should check if: `SPV_EXT_demote_to_helper_invocation` & `DemoteToHelperInvocationEXT` is enabled